presto-native-execution/presto_cpp/main/CMakeLists.txt

```diff
@@ -121,3 +121,23 @@ if(PRESTO_STATS_REPORTER_TYPE)
         "${PRESTO_STATS_REPORTER_TYPE} is not a valid stats reporter name")
   endif()
 endif()
+
+set(PRESTO_MEMORY_CHECKER_TYPE
```

Comment:
You don't need this because the enablement for CI doesn't happen in the makefiles.